src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseExportService.java

```diff
@@ -613,7 +613,7 @@ private File createZipWithAllRepositories(ProgrammingExercise programmingExercis
      * @return The checked out and zipped repository
      * @throws IOException if zip file creation failed
      */
-    private Path createZipForRepositoryWithParticipation(final ProgrammingExercise programmingExercise, final ProgrammingExerciseStudentParticipation participation,
+    public Path createZipForRepositoryWithParticipation(final ProgrammingExercise programmingExercise, final ProgrammingExerciseStudentParticipation participation,
```

Comment:
For reviewers: This is needed for the `AthenaRepositoryExportService`, which needs to be able to zip a single repository.